packages/preview-middleware/src/base/flp.ts

```diff
@@ -196,20 +196,23 @@ export class FlpSandbox {
             },
             locateReuseLibsScript: this.config.libs ?? (await this.hasLocateReuseLibsScript())
         };
+        const id = manifest['sap.app'].id;
         this.addApp(manifest, {
             componentId,
-            target: resources[componentId ?? manifest['sap.app'].id] ?? this.templateConfig.basePath,
+            target: resources[componentId ?? id] ?? this.templateConfig.basePath,
             local: '.',
             intent: this.config.intent
         });
         this.addStandardRoutes();
         if (this.rta) {
+            // const appTitle = manifest['sap.app'].title;
```

Comment:
Please delete this comment

Reply:
Removed: https://github.com/SAP/open-ux-tools/pull/1553/commits/d883f00d3851cef9b527d9a6f829bce6714ddb34